Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM alpine
+
+ARG VIPS_VERSION
+
+RUN apk update
+
+RUN apk add build-base pkgconfig glib-dev gobject-introspection-dev expat-dev tiff-dev libjpeg-turbo-dev libexif-dev giflib-dev librsvg-dev lcms2-dev libpng-dev orc-dev libwebp-dev libheif-dev
+
+RUN apk add libimagequant-dev --repository=http://dl-cdn.alpinelinux.org/alpine/edge/testing
```

Comment:
libimagequant-dev is GPL - are we OK with this?

Reply:
I don't know about this... if you think it should not be there, I can remove

Reply:
We'd be distributing (by making a docker image available) a binary in which GPL software was linked to non-GPL software, so it would be a GPL licence violation.

Users who want to include GPL software in their stack need to build the stack themselves. I think (??) distributing a docker image which builds a stack which violates the GPL is not a GPL violation, so we could do that.

Reply:
Oh hmm that's not quite correct. Including GPL packages just makes the whole thing GPL, so any downstream users of the docker image would be GPL too. Many commercial users would find that unacceptable.

Reply:
I'll remove then